cli/cluster/cmd/common.go

```diff
@@ -184,65 +184,75 @@ func connectEtcd(uriOpts UriOpts, connOpts connectOpts) (*clientv3.Client, error
 	return etcdcli, nil
 }
 
-// createPublisher creates a new data publisher and collector based on UriOpts.
+// selectStorage determines a storage based on the opts.
+func selectStorage(connOpts connectOpts, opts UriOpts,
```

Comment:
It seems not like a select storage, but a do something with a storage connection.
```suggestion
// doOnStorage determines a storage based on the opts.
func doOnStorage(connOpts connectOpts, opts UriOpts,
```
```suggestion
// doStorage determines a storage based on the opts.
func doStorage(connOpts connectOpts, opts UriOpts,
```

Reply:
The first suggestion is applied. Thank you!